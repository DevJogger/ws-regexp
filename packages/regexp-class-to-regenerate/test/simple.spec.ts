import regexpClassToObject from '../index';

test(`regexpClassToObject`, () =>
{

	let actual = regexpClassToObject(/[\u200b-\u200f\ufeff\u2060]/ug)
		.remove(0x200d);

	let re = actual.toRegExp();

	expect(re).toBeInstanceOf(RegExp);

	expect(actual).toMatchSnapshot();
	expect(re).toMatchSnapshot();

});

