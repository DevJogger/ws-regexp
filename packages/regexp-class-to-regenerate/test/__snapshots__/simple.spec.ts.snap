// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`regexpClassToObject 1`] = `
regenerate {
  "data": Array [
    8203,
    8205,
    8206,
    8208,
    8288,
    8289,
    65279,
    65280,
  ],
}
`;

exports[`regexpClassToObject 2`] = `/\\[\\\\u200B\\\\u200C\\\\u200E\\\\u200F\\\\u2060\\\\uFEFF\\]/gu`;
